### ISSUE TITLE
wwwadmin site should have "logout" link

### ISSUE THREAD

**Issue by adamrp** [qiime/qiime_web_app#107]
Currently http://www.microbio.me/wwwadmin has no logout link.  Closing the tab is not safe, because undoing the close-tab brings you back into the admin portal.

**Comment by ackermag** [comment]
Did I close the tab?

On Wed, Jan 16, 2013 at 4:38 PM, adamrp notifications@github.com wrote:

> Currently http://www.microbio.me/wwwadmin has no logout link. Closing the
> tab is not safe, because undoing the close-tab brings you back into the
> admin portal.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/qiime/qiime_web_app/issues/107.

## 

Gail Ackermann
Knight Lab UCB 596
University of Colorado
Boulder

**Comment by adamrp** [comment]
No, this is a general security issue, not something that one of us did wrong.  The issue is that once you login to the wwwadmin site, there is no way to log out.  If you close the tab/window, your logged-in session can be recovered by "unclosing" the tab (apple-shift-t on mac, ctrl-shift-t on windows)

**Comment by douginator2000** [comment]
Will you log an issue on GitHub?

On Wednesday, January 16, 2013, adamrp wrote:

> No, this is a general security issue, not something that one of us did
> wrong. The issue is that once you login to the wwwadmin site, there is no
> way to log out. If you close the tab/window, your logged-in session can be
> recovered by "unclosing" the tab (apple-shift-t on mac, ctrl-shift-t on
> windows)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/qiime/qiime_web_app/issues/107#issuecomment-12347677.

## 

Doug Wendel
Professional Research Associate
University of Colorado at Boulder
Department of Chemistry and Biochemistry, Knight Lab
303-641-5442
wendel@colorado.edu

**Comment by ElDeveloper** [comment]
![33512136](https://f.cloud.github.com/assets/375307/74037/da5413ca-6072-11e2-8a63-adb7f15005f5.jpg)

:P

**Comment by adamrp** [comment]
Where does this code live?  It is not in the qiime_web_app repo; is it on SVN or something?  I added the log out link to the code on www, but should probably change the repo version, wherever that is...
